subsys/bluetooth/services/ble_hids/Kconfig

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+menuconfig BLE_HIDS
+	bool "HID service"
+	depends on SOFTDEVICE_PERIPHERAL
+
+if BLE_HIDS
+
+config BLE_HIDS_BOOT_MOUSE
+	bool "Add boot mouse input characteristic"
+	help
+	  Add the boot mouse input characteristic.
+
+config BLE_HIDS_BOOT_KEYBOARD
+	bool "Add boot keyboard characteristics"
+	help
+	  Add the boot keyboard input and output characteristics.
+
+choice
+	prompt "Default HID protocol mode"
+	depends on BLE_HIDS_BOOT_KEYBOARD || BLE_HIDS_BOOT_MOUSE
+
+config BLE_HIDS_PROTOCOL_MODE_REPORT
+	bool "Report"
+
+config BLE_HIDS_PROTOCOL_MODE_BOOT
+	bool "Boot"
+
+endchoice
+
+config BLE_HIDS_DEFAULT_PROTOCOL_MODE
+	int
+	default 0 if BLE_HIDS_PROTOCOL_MODE_BOOT
+	default 1 if BLE_HIDS_PROTOCOL_MODE_REPORT
+
+config BLE_HIDS_INPUT_REPORT_MAX_NUM
+	int "Number of input reports"
+	range 0 10
+	default 10
+	help
+	  Number of input reports
+
+config BLE_HIDS_INPUT_REPORT_MAX_LEN
+	int "Maximum input report length" if (BLE_HIDS_INPUT_REPORT_MAX_NUM != 0)
+	default 0 if (BLE_HIDS_INPUT_REPORT_MAX_NUM = 0)
+	default 8
+
+config BLE_HIDS_OUTPUT_REPORT_MAX_NUM
+	int "Number of output reports"
+	range 0 10
+	default 10
+	help
+	  Number of output reports
```

Comment:
Nit: I would add a full stop at the end of all of the help notes that are missing it.